impl/pom.xml

```diff
@@ -427,18 +431,44 @@
         </profile>
 
         <profile>
-            <!-- auto-activated profile for any JDK 9+ -->
-            <id>jdk9+</id>
-            <activation>
-                <jdk>[9,)</jdk>
-            </activation>
-            <dependencies>
-                <dependency>
-                    <groupId>com.sun.xml.ws</groupId>
-                    <artifactId>jaxws-rt</artifactId>
-                    <version>3.0.0-M3</version>
-                </dependency>
-            </dependencies>
+            <!-- Run this profile twice to verify the two pass tests under
+            org.jboss.cdi.tck.test.spi.*
+             -->
+            <id>twopass-tests</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludedGroups>cdi-full,integration,javaee-full,se</excludedGroups>
+                            <basedir>${project.basedir}</basedir>
+                            <!-- First load from the annotation processor output directory -->
+                            <testClassesDirectory>target/test-classes2</testClassesDirectory>
+                            <additionalClasspathElements>target/test-classes</additionalClasspathElements>
+                            <systemPropertyVariables>
+                                <!--arquillian.launch>arc</arquillian.launch-->
+                            </systemPropertyVariables>
+                            <forkMode>once</forkMode>
+                            <skipAfterFailureCount>2500</skipAfterFailureCount>
+                            <suiteXmlFiles>
+                                <suiteXmlFile>target/test-classes/twopass-suite.xml</suiteXmlFile>
+                            </suiteXmlFiles>
+
+                            <properties>
+                                <property>
+                                    <name>usedefaultlisteners</name>
+                                    <value>false</value>
+                                </property>
+                                <property>
+                                    <name>surefire.testng.verbose</name>
+                                    <value>10</value>
+                                </property>
+                            </properties>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
```

Comment:
Just curious, this isn't really needed here, is it? This will have to be done by _consumers_ of the TCK, right?

Reply:
Yes, that is used by the impl/src/test tree in the org.jboss.cdi.tck.test.spi.* packages to have some validation of the two pass/annotation processor behavior.